CMakeLists.txt

```diff
@@ -123,7 +123,18 @@ function(buildSdk)
         set (CONNECT_BUILD_TYPE ${CMAKE_BUILD_TYPE})
     endif ()
 
-    set(SDK_VER "${ConnectSDK_VERSION_MAJOR}.${ConnectSDK_VERSION_MINOR}.${ConnectSDK_VERSION_REVISION}")
+    if (DEFINED ENV{BUILD_NUMBER})
+        set (CONNECT_BUILD_NUMBER $ENV{BUILD_NUMBER})
+    else ()
+        message (WARNING "BUILD_NUMBER env variable not set, using value 0 for build number")
+        set (CONNECT_BUILD_NUMBER 0)
```

Comment:
We can. But 'custom' isn't a number and will look weird 01.00.001.custom, so I chose to use invalid build number resulting in 01.00.001.00000